x-pack/plugin/migrate/src/main/java/org/elasticsearch/xpack/migrate/action/CreateIndexFromSourceAction.java

```diff
@@ -30,18 +35,35 @@ private CreateIndexFromSourceAction() {
         super(NAME);
     }
 
-    public static class Request extends ActionRequest implements IndicesRequest {
-
+    public static class Request extends ActionRequest implements IndicesRequest, ToXContent {
         private final String sourceIndex;
         private final String destIndex;
-        private final Settings settingsOverride;
-        private final Map<String, Object> mappingsOverride;
+        private Settings settingsOverride = Settings.EMPTY;
+        private Map<String, Object> mappingsOverride = Map.of();
+        private static final ParseField SETTINGS_OVERRIDE = new ParseField("settings_override");
+        private static final ParseField MAPPINGS_OVERRIDE = new ParseField("mappings_override");
```

Comment:
A very minor niggle and not sure what our standard is but suffixing the names of these with `_FIELD` might make it more obvious how these differ from the actual override class data fields as it tripped me up for a second or two.